arcane/src/arcane/mesh/GhostLayerBuilder.cc

```diff
@@ -486,14 +484,23 @@ _exchangeCells(HashTableMapT<Int32,SharedArray<Int32>>& cells_to_send,bool with_
     Int32 rank = sm->destination().value();
     ISerializer* s = sm->serializer();
     Int32ConstArrayView items_to_send = cells_to_send[rank];
-    m_mesh->serializeCells(s,items_to_send);
```

Comment:
Be careful, by doing this you prevent from calling _serializeItems  (cf DynamicMesh::serializeCells) or _deserializeItems (cf DynamicMesh::addCells(ISerializer) ) when a FamilyNetwork is used and activated.

Reply:
The problem is that addCell and serializeCells is deprecated in IMesh and we do not have UnitTests illustrating the use of network activated to manage the relation between cells, nodes faces and edges...
It is difficult to maintain cell connectivity based on the network without UnitTests